zerver/tests/test_docs.py

```diff
@@ -321,6 +321,7 @@ class IntegrationTest(ZulipTestCase):
     def test_check_if_every_integration_has_logo_that_exists(self) -> None:
         for integration in INTEGRATIONS.values():
             path = urlsplit(integration.logo_url).path
+            assert isinstance(path, str)
```

Comment:
mypy is worried that `integration.logo_url` might be `None`, thereby hitting the second of the two [`urlsplit` overloads](https://github.com/python/typeshed/blob/5598b49851579e09873095845ad6c6c612b07800/stdlib/urllib/parse.pyi#L166-L169) that gives `bytes` instead of `str`. We should `assert integration.logo_url is not None` instead. (And perhaps we should tighten its type and move the assertions back?)

Reply:
Hmm, optimally we can move this assertion to `Integration` so that `logo_url` will never be `None`, but this is not the case for `EmbeddedBotIntegration`.